src/shared/compatibility.js

```diff
@@ -172,6 +172,13 @@ PDFJS.compatibilityChecked = true;
   Number.isInteger = require('core-js/fn/number/is-integer');
 })();
 
+(function checkStringStartsWith() {
```

Comment:
As mentioned in https://github.com/mozilla/pdf.js/pull/9345#issuecomment-357431768, please implement a solution not requiring a polyfill and remove any changes to this file.

(Also, the polyfill is wrong since it should've been `String.prototype.startsWith`, but that's beside the point here since the polyfill isn't necessary.)